README.md

```diff
@@ -117,36 +125,24 @@ If you do not have `rspec` or `rubocop` installed locally, run
 `./scripts/start-docker-for-testing.sh` and execute the commands in the docker
 container. Prepend "sudo" to the script if you are an unprivileged user.
 
-
-
 #### <a name="running-unit-and-integration-tests"></a> Running Unit and Integration Tests
 
-##### In a Docker container
+##### With Docker
 
-* Run tests using the script provided. This script pulls a docker image and runs
-  the tests within a container because integration tests require Linux specific
-  features.
+Running tests for this release requires Linux specific setup and it takes advantage of having the same configuration as Concourse CI, so it's recommended to run the tests (units & integration) in docker containers.
 
-  Notice/warning: the script is called `run-unit-tests-in-docker` but it really
-  runs unit *and* integration tests, that's why they need to run in a container.
+1. `./scripts/docker/container.bash <mysql-8.0(or mysql),mysql-5.7,postgres>`: This will create a docker container with appropriate mounts.
+1. `/repo/scripts/docker/build-binaries.bash`: This will build binaries required for running tests e.g. nats-server
 
-  ```bash
-  ./scripts/run-unit-tests-in-docker #sudo for unprivileged users
-  ```
+- `/repo/scripts/docker/test.bash`: This will run all tests in this release
+- `/repo/scripts/docker/test.bash gorouter`: This will only run `gorouter` tests
+- `/repo/scripts/docker/test.bash gorouter router`: This will only run `router` sub-package tests for `gorouter` package
 
-* If you'd like to run a specific component's tests in a Docker container,
-  the `run-unit-tests` script also takes a package name as an argument:
+The following helper functions are additionally available by sourcing `./scripts/docker/helpers.bash`:
 
-  ```bash
-  ./scripts/run-unit-tests-in-docker gorouter
-  ```
+- `docker_run_routing_release_tests <mysql-8.0(or mysql),mysql-5.7,postgres>`:  This function combines steps described above sequentially.
```

Comment:
Can we bring back a dockerized run-template-tests helper function/script?

Reply:
I think `./scripts/template-tests` is still part of the repo and can be executed inside/outside of the docker container.